rtt/ServiceRequester.cpp

```diff
@@ -67,7 +67,7 @@ namespace RTT
             log(Error) << "OperationCaller with name '" + isb.getName() + "' already present." << endlog();
             return false;
         }
-        mmethods.insert(make_pair<std::string, OperationCallerBaseInvoker*> (isb.getName(), &isb));
+        mmethods.insert(make_pair<std::string, OperationCallerBaseInvoker*> (std::string(isb.getName()), &isb));
```

Comment:
I'm not sure will try next week, the first line is the original line, i compile also with clang (not only cxx11) the problem was that getName returns a const string, and the template is a non-const string which causes the compilation to fail.

Nevertheless i will try your second proposal